.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,40 @@
-**IMPORTANT**: PRs without the template will be CLOSED
-
-### Description of your *pull request* and other information
-
 <!--
-
-Explanation of your *pull request* in arbitrary form goes here. Please **make sure the description explains the purpose and effect** of your *pull request* and is worded well enough to be understood. Provide as much **context and examples** as possible
-
+    **IMPORTANT**: PRs without the template will be CLOSED
+    
+    Due to the high volume of pull requests, it may be a while before your PR is reviewed.
+    Please try to keep your pull request focused on a single bugfix or new feature.
+    Pull requests with a vast scope and/or very large diff will take much longer to review.
+    It is recommended for new contributors to stick to smaller pull requests, so you can receive much more immediate feedback as you familiarize yourself with the codebase.
+
+    PLEASE AVOID FORCE-PUSHING once a review has started, as it makes reviewing more difficult.
```

Comment:
I've lost draft review as well
```suggestion
    PLEASE AVOID FORCE-PUSHING once a PR is opened, as it makes reviewing more difficult.
```

Reply:
Maybe this?

```suggestion
    PLEASE AVOID FORCE-PUSHING once you opened a PR, as it makes reviewing more difficult.
```

Contributors see this notification when they are opening a new PR.